3rdparty/jvm/com/twitter/BUILD

```diff
@@ -14,3 +14,9 @@ jar_library(name='scrooge-core',
     scala_jar(org='com.twitter', name='scrooge-core', rev=SCROOGE_REV),
   ],
 )
+
+jar_library(name='scalac-scoverage-plugin',
+  jars=[
+    scala_jar(org='com.twitter.scoverage', name='scalac-scoverage-plugin', rev='1.0.1-twitter'),
```

Comment:
Please include a link to the github repository of the fork, the PR that the fork contains, and a bit of explanation of what we should do to resolve the fork (ie, periodically ping upstream on that PR).

Also, since this is named `com.twitter.scoverage`, should put it in a subdirectory probably: `3rdparty/jvm/com/twitter/scoverage/BUILD`